teamcity/flake8_v3_plugin.py

```diff
@@ -0,0 +1,47 @@
+import re
+
+from flake8.formatting import base
+
+from teamcity.messages import TeamcityServiceMessages
+from teamcity import __version__
+
+
+class TeamcityReport(base.BaseFormatter):
+    name = 'teamcity-messages'
+    version = __version__
+    messages = TeamcityServiceMessages()
+
+    options_added = False
+
+    @classmethod
+    def add_options(cls, parser):
+        if not cls.options_added:
+            parser.add_option('--teamcity', default=False,
```

Comment:
Ah that helps, but wouldn't it still require passing  `--format=teamcity-messages`? Can we make this formatted the default formatter when TeamCity is detected?